### PR TITLE
Dev > Master

### PR DESCRIPTION
**Pull request by drpatelh**
Many thanks to contributing to nf-core/atacseq!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [ ] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/atacseq branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/atacseq)
 - [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [x] Make sure your code lints (`nf-core lint .`).
 - [x] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [x] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/atacseq/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by drpatelh** [comment]
Thanks @MaxUlysse !! Just need one more approval on this and we are ready to go ✈️

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,57 @@
 # nf-core/atacseq: Changelog
 
-## v1.0dev - [date]
-Initial release of nf-core/atacseq, created with the [nf-core](http://nf-co.re/) template.
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [1.1.0] - 2019-11-01
```
Comment (apeltzer) [inline CHANGELOG.md]:
```suggestion
## [1.1.0] - 2019-11-05
```

Maybe 5th?

File: bin/check_design.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 #######################################################################
 #######################################################################
```
Comment (apeltzer) [inline bin/check_design.py]: I think I'll get this going for Eager soon too 👍 Looks fab :-)

File: CHANGELOG.md (hunk 1 above)
Comment (drpatelh) [reply CHANGELOG.md]: Ah, that's been changed in the open PR to `dev` :+1:

Comment (drpatelh) [reply CHANGELOG.md]: I'll merge that last minute before the merge to `master`

File: bin/check_design.py (hunk 2 above)
Comment (drpatelh) [reply bin/check_design.py]: Thanks. It does work really well, restricts user error and gives you experiment-level info :smiley:

**Pull request merged**